simulation/run_simulation.py

```diff
@@ -292,7 +303,34 @@ def run_unoptimized_and_export(input_speed=None, values=None, race_type="ASC", g
     return results_array
 
 
+def run_hyperparameter_search(simulation_model: Simulation, bounds: InputBounds):
```

Comment:
add a doc string for this function to explain how to use it and maybe rename the function since it doesn't search but tests different settings and writes the results for each